components/ResizeableWindow.jsx

```diff
@@ -169,7 +170,10 @@ class ResizeableWindow extends React.Component {
             "resizeable-window-body-scrollable": this.props.scrollable,
             "resizeable-window-body-nonscrollable": !this.props.scrollable
         });
-        const style = {display: this.props.visible ? 'initial' : 'none'};
+        const style = {
+            display: this.props.visible ? 'initial' : 'none',
+            right: 'calc(0.25em + ' + this.props.menuMargins.right + 'px)'
```

Comment:
What is 0.25em? Shouldn't this be part of the menuMargin, if necessary? Else with a zero menuMargin, it will still have a 0.25em right offset.
You'll also want to honour the left margin, since you support it in the reducer state.

Reply:
I add 0.25em as it is done for MapCopyright : 

https://github.com/qgis/qwc2/blob/master/plugins/MapCopyright.jsx#L57

I found it aesthetically pleasing when everything wasn't aligned (stuck) with the right edge; even without compact menu.

I hope to honor the left margin but I can't do this right now. Should I remove left margin from reducer state ?